goodtablesio/utils/backend.py

```diff
@@ -0,0 +1,42 @@
+# pylama:ignore=E301
+import datetime
+from werkzeug.http import http_date
+from flask import make_response, request
+from functools import wraps, update_wrapper
+from goodtablesio.models.user import User
+
+
+# Module API
+
+def no_cache(view):
+    @wraps(view)
+    def no_cache(*args, **kwargs):
+        http_now = http_date(datetime.datetime.now())
+        response = make_response(view(*args, **kwargs))
+        response.headers['Last-Modified'] = http_now
+        response.headers['Expires'] = http_now
+        response.headers['Cache-Control'] = 'no-cache'
+        response.headers['Pragma'] = 'no-cache'
+        return response
+    return update_wrapper(no_cache, view)
+
+
+def token_required(view):
+    @wraps(view)
+    def decorated_view(*args, **kwargs):
+        token = request.headers.get('Authorization')
```

Comment:
I wonder if we should use the form `Authorization: <type> <credentials>` (ie `Authorization: token xxxx`) to align better with other common implementations and in case we want to support other auth mechanisms in the future. It should be an easy change.
WDYT?